testing/src/main/java/org/oppia/android/testing/profile/ProfileTestHelper.kt

```diff
@@ -105,6 +105,12 @@ class ProfileTestHelper @Inject constructor(
    */
   fun logIntoNewUser() = logIntoProfile(internalProfileId = 2)
 
+  /**
+   * Login to a new user profile that has no progress for any topics or stories. This relies on other
+   * tests utilizing profile 1 as the default user profile so that profile 2 never has any progress.
+   */
+  fun loginToNewUser() = logIntoProfile(internalProfileId = 2)
```

Comment:
this gets duplicate here. please check the code below 108-112 line number.

Reply:
I had to update this code with develop. Its corrected now.